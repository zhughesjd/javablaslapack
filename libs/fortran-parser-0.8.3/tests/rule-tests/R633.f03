! Test nullify-stmt, which is:
!      nullify-stmt  is  NULLIFY ( pointer-object-list )
!
! Tested separetly is: pointer-object-list (R634).
nullify(a)
nullify(a%b)

end
