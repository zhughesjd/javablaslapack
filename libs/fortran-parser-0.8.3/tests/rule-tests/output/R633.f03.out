*******************************************
args[1]: R633.f03
R1101:main-program__begin:
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=0
R 634:pointer-object-list__begin:
R 613:part-ref: id=[@10,149:149='a',<209>,5:8] hasSelectionSubscriptList=false hasImageSelector=false
R 612:data-ref: numPartRef=1
R 634:pointer-object:
R 634:pointer-object-list: count=1
R 633:nullify-stmt:
R 214:action-stmt:
R 213:executable-construct:
R 634:pointer-object-list__begin:
R 613:part-ref: id=[@15,160:160='a',<209>,6:8] hasSelectionSubscriptList=false hasImageSelector=false
R 613:part-ref: id=[@17,162:162='b',<209>,6:10] hasSelectionSubscriptList=false hasImageSelector=false
R 612:data-ref: numPartRef=2
R 634:pointer-object:
R 634:pointer-object-list: count=1
R 633:nullify-stmt:
R 214:action-stmt:
R 213:executable-construct:
R 209:execution-part-construct:
R 208:execution-part:
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=true hasInternalSubprogramPart=false
Parser exiting normally
********************************************
